robot_specific_config/abb_irb4600_support/urdf/irb4600_40_255_macro.xacro

```diff
@@ -45,6 +54,9 @@
         </geometry>
         <xacro:material_abb_graphite_white/>
       </visual>
+      <xacro:cylinder_inertial radius="0.06" length="1.32" mass="34.32">
+        <origin xyz="0 0 0.65" rpy="0 0 0" />
```

Comment:
where did this value for the `z` come from?

Reply:
also from solidworks